### ISSUE TITLE
CLI tool: return non-zero exit code when warnings are found

### ISSUE THREAD

**Issue by Kelvin4702** [fsprojects/FSharpLint#339]
### Description

The dotnet-fsharplint CLI tool currently returns a 0 exit code, regardless of whether lint warnings were found. It would be nice to be able to determine when something is deemed successful through the exit code.

### Repro steps

1. Run the CLI tool on a file with warnings: observe a 0 exit code.
2. Run the CLI tool on a file without warnings: observe a 0 exit code.

### Expected behavior

A success (0) exit code is returned when no warnings are found in the lint run.
A non-zero exit code is returned when lint warnings are found.

### Actual behavior

CLI runs that produce warnings, and CLI runs that produce no warnings both return a 0 exit code.

### Known workarounds

Piping the output into another tool, i.e. `grep` to determine if a run was successful, i.e.
```
dotnet fsharplint -f "foo.fsproj" | grep "Summary: 0 warnings" > /dev/null
```
This works because grep returns a 0 exit code if the search string was found, and a 1 exit code if the search string was not found.

### Related information 

* Operating system: Windows 10 Pro
* dotnet-fsharplint version: 0.11.1
* .NET Core version: 2.1.505
